.github/ISSUE_TEMPLATE/bug_repor

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
`bug_repor[t.md]`

Reply:
HA, when you fat finger a file name in emacs

Reply:
I almost blindly approved it since the content was right, but the fact that it was not highlighted as markdown made me second guess. Then I thought GH was truncating so I had to go look at another one of your PRs to verify. 😆